src/ert/scheduler/scheduler.py

```diff
@@ -270,9 +267,6 @@ async def _publisher(self) -> None:
                 while True:
                     if event is None:
                         event = await self._events.get()
-                    if event == CLOSE_PUBLISHER_SENTINEL:
-                        self._publisher_done.set()
```

Comment:
You are removing all usage of _publisher_done, so it should probably be removed from the class as well.

Reply:
Thanks! I thought I did remove it. It might have got lost in the rebasing.